### DISCUSSION TITLE
WLED ESP32 Multiple GPIO support

### DISCUSSION THREAD

**Discussion by CPWDEV** [awawa-dev/HyperHDR#1021] [General]
Good Morning,

I have HyperHDR running behind my tv ( ESP32 SK6812 RGBW ). I added a second GPIO out to run a second a set of LEDs under my tv stand. I can't find a way to set the second GPIO as a separate instance to only mirror the bottom of the TV. I am currently using a second ESP32 to control the tv stand lights now, but want to take that out if possible. 

If it is not possible that's fine i have a working setup but i am interested if there is another solution.

Thanks,
Chris

**Comment by awawa-dev** [comment]
Hi
Depending on the setup design, cable lengths etc, assuming both LED strips are the same type, this can even be achieved using a single instance with single mode (both LEDs connected as one) and multi-segment mode (separate GPIO for each LED strip).

**Reply by CPWDEV** [reply]
How can I control the separate GPIO output in HyperHDR? Right now my WLED has segments for the tv and tv stand but I can’t figure out how to make the tv stand segment the same as the bottom of my tv

**Reply by awawa-dev** [reply]
I don't use or support WLED here, we have our own recommended LED drivers. But I assume WLED can set up a virtual LED strip using different outputs and combine them into one. Then from a HyperHDR perspective it's one continuous LED strip.

**Reply by CPWDEV** [reply]
yes I can set it up as one continuous strip, but in HyperHDR i can't figure out how you would loop the bottom row so that it then displays twice. The bottom section of my tv has 84 LEDs and my TV Stand has 91 LEDs. If i set the bottom to 175, all LEDs light up but the bottom row density is just 175 instead of a row of 84 and a row of 91

**Reply by awawa-dev** [reply]
It's just a matter of planning the layout correctly. I suggest starting with a layout just for TV in the first instance, but so that it ends in one of the corners at the bottom. In the second temporary instance, design a layout for the bar at the bottom so that it starts in the corner where the first instance ended (although this depends on the cable connections). Copy this layout as JSON to the clipboard and paste the layout in the first instance at the end to connect them. Then delete the second instance.

**Reply by CPWDEV** [reply]
that makes a lot of sense and that is how my current layout starts/ends i will play around with that thank you

**Reply by Oli-70** [reply]
Hello, has anyone found a solution? I can't get the temporary profile to work as awawa says, only controlling the last LEDs. In wled, the individual light strips at the individual connections are recognized as continuous light strips, which are only separated by segments. There is an option called "Custom bus start indices" in wled, but HyperHDR doesn't seem to understand this. So I would need to be able to address LEDs 300-400 in the temporary profile, for example.
